src/citrine/_serialization/properties.py

```diff
@@ -325,6 +325,41 @@ def _serialize(self, value: list) -> list:
         return serialized
 
 
+class Set(Property[set, typing.Iterable]):
```

Comment:
This seems cool, but a list would be fine if you end up running into any serde issues trying to support this